toqito/matrix_props/is_permutation.py

```diff
@@ -0,0 +1,71 @@
+"""Is matrix a permutation matrix."""
+import numpy as np
+
+def is_permutation(mat: np.ndarray) -> bool:
+    r"""
+    Determine if a matrix is a permutation matrix [WikSquare]_.

```

Comment:
You should change the reference tag `[WikSquare]` to something that is more descriptive of the task (like, say, `[WikPermMat]` or something to that effect.

Reply:
Great catch!   Done.